dockerfiles/e2e/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM library/centos:centos7
+
+ENV LANG=en_US.utf8 \
+    DISPLAY=:99 \
+    FABRIC8_USER_NAME=fabric8
+
+COPY google-chrome.repo /etc/yum.repos.d/google-chrome.repo
+COPY e2e /root/e2e
+RUN yum install --assumeyes epel-release && \
```

Comment:
I would move all yum commands before the copy of files to keep Docker layers the same

Then copy only package*.json and run npm install and then copy all other e2e files